cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -0,0 +1,388 @@
+/*
```

Comment:
ok, we can fix that by making raw namer take an additional arg, of the package it's in. Then each generator that needs this can construct the raw namer with this arg. I can make this change, but probably won't have time for a few days.